Sources/nuklear/include/nuklear_phiar.h

```diff
@@ -0,0 +1,10 @@
+#define NK_INCLUDE_FIXED_TYPES
+#define NK_INCLUDE_STANDARD_IO
+#define NK_INCLUDE_STANDARD_VARARGS
+#define NK_INCLUDE_DEFAULT_ALLOCATOR
+#define NK_INCLUDE_VERTEX_BUFFER_OUTPUT
+#define NK_INCLUDE_FONT_BAKING
+#define NK_INCLUDE_DEFAULT_FONT
+#define NK_LIB
+
+#include "nuklear.h"
```

Comment:
Instead of nuklear_phiar (nuclear fire sounds really catastrophic), how about nuklear_library_stubs.h or nuklear_internal.h?

Reply:
Good idea, changed to nuklear_internal